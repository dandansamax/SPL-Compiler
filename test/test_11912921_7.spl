struct Egg
{
    float weight;
    char color[10];
};

int test_7()
{
    int a = 3.5;
    struct Egg simple_egg;
    simple_egg.bone = 3.5 + a;
    return 0;
}