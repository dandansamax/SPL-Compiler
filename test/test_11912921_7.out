Error type 5 at Line 9: unmatching types appear at both sides of the assignment operator (=): LHS is int, but RHS is float.
Error type 14 at Line 11: accessing an undefined structure member: bone
Error type 7 at Line 11: unmatching operands: LHS is float, but RHS is int.
